cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,113 @@
+import PageObject from '../PageObject';
+
+class SettingsPage extends PageObject {
+  url = '/#/settings';
+
+  get usernameField() {
+    return cy.get('[data-qa="new-your-username"]');
+  }
+
+  get userBioField() {
+    return cy.get('[data-qa="new-short-bio"]');
+  }
+
+  get emailField() {
+    return cy.get('[data-qa="new-user-email"]');
+  }
+
+  get newPasswordField() {
+    return cy.get('[data-qa="new-user-password"]');
+  }
+
+  get updateSettingsBtn() {
+    return cy.get('[data-qa="new-btn-update"]')
+  }
+
+  get settingsLink() {
+    return cy.get(':nth-child(3) > .nav-link');
+  }
+
+  get logOutBtn() {
+    return cy.get('[class="btn btn-outline-danger"]');
+  }
+
+  get modalBio() {
+    return cy.get('[class="swal-modal"]');
+  }
+
+  get modalBtnOk() {
+    return cy.get('[class="swal-button swal-button--confirm"]');
+  }
+
+  visit() {
+    cy.visit(this.url);
+  }
+
+  clickOnOkBtn() {
+    this.modalBtnOk.click();
+  }
+
+  assertUpdatiedBio() {
+    this.modalBio
+    .should('contain', 'Update successful!');
+  }
+
+  assertUpdatiedEmail() {
+    this.modalBio
+    .should('contain', 'Update successful!');
+  }
+
+  assertUpdatiedPassword() {
+    this.modalBio
+    .should('contain', 'Update successful!');
+  }
+
+  assertUpdatiedShortPassword() {
+    this.modalBio
+    .should('contain', 'Update failed!');
+  }
```

Comment:
```suggestion
  assertUpdatiedPassword() {
    this.modalBio
    .should('contain', 'Update successful!');
  }

  assertUpdatiedShortPassword() {
    this.modalBio
    .should('contain', 'Update failed!');
  }
```
this is example of the same methods but with different names, its a bad practice, need to create 1 reusable method